javascripts/discourse/controllers/gif.js.es6

```diff
@@ -50,7 +56,10 @@ export default Controller.extend(ModalFunctionality, {
 
       $.ajax({ url: this.getEndpoint(this.query, this.offset) })
         .done((response) => {
-          const images = response.data.map((gif) => ({
+          let images;
```

Comment:
Nitpick: indenting seems off in this section.

Reply:
Thanks, apologies, I pushed an additional commit.